macro/src/main/scala/monocle/macros/internal/Macro.scala

```diff
@@ -85,8 +84,8 @@ private[macros] class MacroImpl(val c: blackbox.Context) {
         override def set(a: $bTpe): $sTpe => $tTpe =
           _.copy($field = a)
 
-        override def modifyF[$F[_]: Functor](f: $aTpe => $F[$bTpe])(s: $sTpe): $F[$tTpe] =
-          Functor[$F].map(f(s.$fieldMethod))(a => s.copy($field = a))
+        override def modifyF[$F[_]: cats.Functor](f: $aTpe => $F[$bTpe])(s: $sTpe): $F[$tTpe] =
+          cats.Functor[$F].map(f(s.$fieldMethod))(a => s.copy($field = a))
```

Comment:
For [macro hygiene](https://docs.scala-lang.org/overviews/quasiquotes/hygiene.html) wouldn't it be better if this were fully qualified, i.e. `_root_.cats.Functor`? I guess the same applies to the import of `monocle.PLens` which should probably be `import _root_.monocle.PLens`.